content/rc/api/examples/manage-subscriptions.md

```diff
@@ -44,26 +44,56 @@ POST "https://[host]/v1/subscriptions"
+    You do not need to pass this field in your API request if you subscribed to Redis Enterprise Cloud through GCP Marketplace.
 
-- **`cloudAccountId`** - Specify a cloud account that is defined for your account.
+- **`cloudAccountId`** - Set a cloud account ID that is connected to your account.
 
-    You can look up cloud account identifiers using the `GET /cloud-accounts` API operation or use `"cloudAccountId": 1` to use internal resources.
+    You can look up cloud account IDs using `GET /cloud-accounts`. To use internal resources, set it to `"cloudAccountId": 1`.
 
-    If you subscribed to Redis Enterprise Cloud through the GCP Marketplace, use the value `1` for this field.
+    If you subscribed to Redis Enterprise Cloud through GCP Marketplace, use `1` for this field.
 
-- The JSON document contains two primary segments: subscription specification and databases specification.
-- When you create a subscription, you must specify one or more databases in the "`databases`" array of the above JSON file.
-- You can [copy-and-paste]({{< relref  "/rc/api/get-started/use-rest-api.md#swagger-user-interface" >}}) the contents of the JSON file into the `POST /subscriptions` operation in the [Swagger UI](https://api.redislabs.com/v1/swagger-ui.html).
+The request JSON body contains two primary segments: subscription specification and databases specification. When you create a subscription, you must specify one or more databases in the "`databases`" array of the above JSON file.
+
+You can [copy-and-paste]({{< relref  "/rc/api/get-started/use-rest-api.md#swagger-user-interface" >}}) the contents of the JSON file into the `POST /subscriptions` operation in the [Swagger UI](https://api.redislabs.com/v1/swagger-ui.html).
 
 {{< note >}}
 The Swagger UI generates default JSON examples for `POST` and `PUT` operations. You can reference these examples and modify them to fit your specific needs and account settings. The examples will fail if used as-is.
 {{< /note >}}
 
-The POST response is a JSON document that contains the `taskId`. You can use `GET /v1/tasks/<taskId>` to track the creation of your subscription.
+The response body contains the `taskId` of the task that is creating the subscription. You can use `GET /v1/tasks/<taskId>` to track the status of this task.
+
+## Update a subscription
+
+Use `PUT /v1/subscriptions/<subscriptionId>` to update a subscription.
+
+```sh
+PUT "https://[host]/v1/subscriptions/<subscriptionId>"
+{
+    "name": "new-subscription-name",
+    "paymentMethodId": <payment_id>
+}
+```
+
+You can only change the following settings with this endpoint:
+- **`name`** - Specify a new name for your subscription.
+
+- **`paymentMethodId`** - Specify a different payment method that is defined for your account.
+
+    You can look up a payment method identifier using the `GET /payment-methods` API operation.
```

Comment:
To find the IDs for payment methods connected to your account, use he `GET /payment-methods` API operation.